e2e/cypress/integration/account_settings/general/main_menu_stays_open_spec.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+import users from '../../../fixtures/users.json';
+
+let townsquareChannelId;
+
+describe('Account Settings > General', () => {
+    before(() => {
+        // # Login and go to /
+        cy.apiLogin('sysadmin');
+        cy.visit('/');
+
+        // # Store channel id for further use
+        cy.getCurrentChannelId().then((id) => {
+            townsquareChannelId = id;
+        });
+    });
+
+    it('AS15009 - Main Menu stays open', () => {
+        // # Click the hamburger button
+        cy.get('#headerInfo').find('button').click({force: true});
+
+        // # Wait for the menu to open
+        cy.wait(TIMEOUTS.TINY);
+
+        // * Menu should be visible
+        cy.get('#sidebarDropdownMenu').find('ul').should('be.visible');
```

Comment:
nit: Please update into a more meaningful "find". Same case in [L41](https://github.com/mattermost/mattermost-webapp/pull/4550/files#diff-fcb898a584d61eae4f4c6531000fba4cR41).
```suggestion
        cy.get('#sidebarDropdownMenu').find('.dropdown-menu').should('be.visible');
```